src/argilla/client/feedback/dataset/local/mixins.py

```diff
@@ -274,6 +278,9 @@ def push_to_argilla(
             if len(self.records) > 0:
                 remote_dataset.add_records(self.records, show_progress)
 
+            _LOGGER.info("\n\t✓ Dataset succesfully pushed to Argilla")
```

Comment:
`\n\t` remove this maybe?
```suggestion
            _LOGGER.info("✓ Dataset succesfully pushed to Argilla")
```